src/carnot/funcs/builtins/regex_ops.cc

```diff
@@ -32,6 +32,7 @@ void RegisterRegexOpsOrDie(udf::Registry* registry) {
   // String contains.
   registry->RegisterOrDie<RegexMatchUDF>("regex_match");
   registry->RegisterOrDie<RegexReplaceUDF>("replace");
+  registry->RegisterOrDie<MatchRegexRule>("match_regex_rule");
```

Comment:
Can we mark this as an internal/unsupported function? Renaming it to `_match_regex_rule` will do.

Reply:
Done!